package.xml

```diff
@@ -0,0 +1,22 @@
+<package format="2">
+  <name>dynamic-graph</name>
+  <version>3.7.3</version>
+  <description>
+    Dynamic graph library
+  </description>
+  <maintainer email="ostasse@laas.fr">Olivier Stasse</maintainer>
+  <license>BSDv2</license>
```

Comment:
There is not such thing as a `BSDv2`. We are using a 2-Clauses BSD License, so for ROS, `BSD` is good.

Reply:
Fixed